### ISSUE TITLE
Set up automatic changelog

### ISSUE THREAD

**Issue by Almad** [paver/paver#192]
The manual changelog is error prone and also contributes to procastinating on releases. 

I have good experiences with semantic-release from JS world, so unless someone has better suggestions, I will set up something like https://github.com/relekang/python-semantic-release

**Comment by craigds** [comment]
As a recommendation, don't try to have a full changelog. Do 'upgrade notes' or something. Be meticulous about including any breaking changes, but that's _mostly_ all you need.

The main thing I (and I suspect a lot of people) use changelogs for is to answer the question: is it safe to update to the new version? A changelog that literally only lists breaking changes is very easy to maintain.

**Comment by Almad** [comment]
Good point. I am maintaining full changelog given that's how it was done before in the project, but breaking changes are probably more readable for everyone :)
